src/main/java/com/tw/go/plugin/task/GoPluginImpl.java

```diff
@@ -161,7 +161,7 @@ String cleanupScript(String scriptValue) {
     }
 
     private int executeScript(String workingDirectory, String shType, String scriptFileName, Map<String, String> environmentVariables) throws IOException, InterruptedException {
-        return executeCommand(workingDirectory, environmentVariables, "/bin/bash", "-euo", "pipefail", "-c", "./" + scriptFileName);
```

Comment:
Is it worth keeping the `"./" + ` in the script file name?

I can't imagine it's super important but it feels safe to keep and I'm guessing its solved some kind of issue in the past.

Reply:
Nope, it's only necessary for `-c`.